applications/openshift/api-server/api_server_audit_log_maxsize/rule.yml

```diff
@@ -6,6 +6,17 @@ title: 'Configure Maximum Audit Log Size'
 
 description: |-
     To rotate audit logs upon reaching a maximum size, edit the API Server pod
+{{%- if product == "ocp4" %}}
+    specification file <tt>/etc/kubernetes/static-pod-resources/configmaps/config/config.yaml</tt> on
+    the master node(s) and set the <tt>maximumFileSizeMegabytes</tt> parameter to
+    an appropriate size in MB. For example, to set it to 100 MB:
+    <pre>
+    "auditConfig":{
```

Comment:
```suggestion
    "auditConfig": {
```

Reply:
Going to keep the original syntax considering indentation was not done in the config.